tools/RazorPageGenerator/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "1.1.0-*",
```

Comment:
Not sure if we'll ever release this as a real thing. Just version it super low so we don't have to deal with this if we ever have to release it to nuget.